src/main/java/io/specto/hoverfly/junit/core/model/Request.java

```diff
@@ -48,13 +48,13 @@
     private RequestType requestType;
 
     @Deprecated
-    public RequestMatcher(String path,
-                          String method,
-                          String destination,
-                          String scheme,
-                          String query,
-                          String body,
-                          Map<String, List<String>> headers) {
+    public Request(String path,
```

Comment:
To distinguish it from the DSL request matcher. I think these are just Hoverfly JSON models, the naming isn't too much a big deal, as they are mainly for conversion between POJO and the JSON.